numpy/core/src/umath/ufunc_object.c

```diff
@@ -243,27 +269,13 @@ _find_array_prepare(PyObject *args, PyObject *kwds,
             obj = PyDict_GetItem(kwds, npy_um_str_out);
```

Comment:
You are on purpose not yet checking whether the `out` argument is a tuple, correct?

Reply:
I'm on purpose just leaving it as it was - I want to push the `out` argument parsing way upstream